tavern/internal/www/src/components/page-wrapper/PageWrapper.tsx

```diff
@@ -21,6 +22,7 @@ const navigation = [
   { name: PageNavItem.createQuest, href: '/createQuest', icon: CommandLineIcon, internal: true },
   { name: PageNavItem.dashboard, href: '/dashboard', icon: PresentationChartBarIcon, internal: true },
   { name: PageNavItem.hosts, href: '/hosts', icon: BugAntIcon, internal: true },
+  { name: PageNavItem.users, href: '/users', icon: UserIcon, internal: true },
```

Comment:
We only want to display this view if a user is an Admin.

Also we should change the endpoint to something like /admin-settings